packages/mdc-chips/chip-set/foundation.ts

```diff
@@ -194,7 +191,7 @@ export class MDCChipSetFoundation extends MDCFoundation<MDCChipSetAdapter> {
     if (index >= 0) {
       this.selectedChipIds_.splice(index, 1);
       const chipIndex = this.adapter_.getIndexOfChipById(chipId);
-      this.adapter_.selectChipAtIndex(chipIndex, false);
+      this.adapter_.selectChipAtIndex(chipIndex, false, true);
```

Comment:
Document these boolean values, It makes it easier to discover what these values are mapped to.

Example:

```ts
foo(/** barParam */ true);
```